code/modules/research/techweb/all_nodes.dm

```diff
@@ -460,7 +460,7 @@
 	display_name = "Advanced Mining Technology"
 	description = "Efficiency Level 127"	//dumb mc references
 	prereq_ids = list("basic_mining", "adv_engi", "adv_power", "adv_plasma")
-	design_ids = list("drill_diamond", "jackhammer", "hypermod", "plasmacutter", "plasmacutter_adv")
+	design_ids = list("drill_diamond", "jackhammer", "hypermod", "plasmacutter_adv")
 	research_cost = 2500

```

Comment:
I know you didn't change this but this is fairly cheap considering adv_plassies make mining trivial

Reply:
???

Reply:
Laymen terms: 2.5K for Advanced Plasma Cutters???